statsmodels/genmod/tests/test_glm.py

```diff
@@ -1450,9 +1448,13 @@ def __init__(self):
 
 
 class TestTweediePower2(CheckTweedie):
-    def __init__(self):
+    @classmethod
+    def setupClass(self):
         from .results.results_glm import CpunishTweediePower2
-        super(TestTweediePower2, self).__init__()
+        from statsmodels.datasets.cpunish import load_pandas
+        self.data = load_pandas()
+        self.exog = self.data.exog[['INCOME', 'SOUTH']]
+        self.endog = self.data.endog
```

Comment:
Ahhh... 

I thought about a super but I couldn't get that to work (in about 2 tries)... Didn't attempt this way. 

Thanks for the tip.